src/Tribe/RSVP.php

```diff
@@ -1207,7 +1212,27 @@ public function front_end_tickets_form( $content ) {
 	 */
 	public function login_required() {
 		$requirements = (array) tribe_get_option( 'ticket-authentication-requirements', array() );
-		return in_array( 'event-tickets_rsvp', $requirements );
+		return in_array( 'event-tickets_rsvp', $requirements, true );
+	}
+
+	/**
+	 * Determine whether to use new RSVP views.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool Whether to use new RSVP views.
+	 */
+	public function use_new_views() {
```

Comment:
Can we make this into a template tag to have more explicit visibility on it? Similar to what we did on Views V2?

Reply:
A global function that has the logic in it? Or a global function that calls this method?

Reply:
Global with the logic in it.